### PR TITLE
Feature/fix schedule

### PR DESCRIPTION
**Pull request by raksiv**
Schedule was not mapping to the right name property, and the topic was looking at ID not name.

### CODE REVIEW

**Review by tjholm — CHANGES_REQUESTED** [review]
Looks great! Should probably remove the manual changelog update (we should think about adding it to the ignore file).

**Comment by raksiv** [comment]
Amended the commit to drop the changelog.

**Review by tjholm — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -106,3 +106,6 @@ lib/
 !**/src/**/*
 
 oclif.manifest.json
+
+# Autogenerated change log
+CHANGELOG.md
```
Comment (davemooreuws) [inline .gitignore:111]: Shouldn't the change-log be pushed during release?

**Review by davemooreuws — CHANGES_REQUESTED** [review]
Looks good to me, not sure about the changelog gitignore though

File: .gitignore (hunk 1 above)
Comment (raksiv) [reply .gitignore:111]: Not sure about this - happy to re-discuss
We currently agreed to keep it in gitignore.

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**